FordLabs/PeopleMover
# 102 assign new owner

## Issue
Resolves 102

## What was done
- [x] Added endpoint to backend for transferring ownership of a space
- [x] Added option to assign ownership in dropdown on "share access" modal
- [x] Removed ownership option in dropdown when current user is not an owner
- [x] Upgraded e2e backend controller to set accessToken cookie for cypress tests   

## How to test
1. If you are the only member of a space, invite another person to be an editor in the space
2. Add #perm to the end of your url, and select and hit enter on the url twice (enables the proper feature toggle)
2. Go the share access modal,  select the dropdown menu for the other user, and click on owner
3. Verify that they are now marked as the owner, and you are shown as an editor
4. Click on your editor dropdown, and verify that the owner option does not appear

## Accessiblity Criteria
### Testing

### Manual Checks
